posters-overrides.json

```
@@ -4578,7 +4578,10 @@
    "number": 1618
   },
   {
-   "number": 1621
+   "number": 1621,
+   "videochat": "https://zoom.us/j/9545579483?pwd=QUM3Um55MWV2RmFuWTJkb3ZKV2xQdz09",
+   "password": "visualqc"
```

Videochat should be HTML snippet, and password doesn't exist as a field. What about: 
```suggestion
   "videochat": "<a href=\"https://zoom.us/j/9545579483?pwd=QUM3Um55MWV2RmFuWTJkb3ZKV2xQdz09\" target=\"_ohbm2020-1621\">zoom:9545579483</a> (password: \"visualqc\")",
```